tests/zero_code_change/test_pytorch_disengage_smdebug.py

```diff
@@ -0,0 +1,58 @@
+# Third Party
+# Standard Library
+import os
+from unittest.mock import patch
+
+import pytest
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from tests.zero_code_change.pt_utils import Net, get_dataloaders
+
+# First Party
+import smdebug.pytorch as smd
+from smdebug.core.config_validator import reset_config_validator
+from smdebug.core.utils import SagemakerSimulator
+
+
+@pytest.fixture(autouse=True)
+def cleanup():
+    yield
+    os.environ.pop("USE_SMDEBUG", None)
+    os.environ.pop("SM_HPS", None)
+    reset_config_validator()
+
+
+@patch("smdebug.core.config_validator.is_framework_version_supported", return_value=False)
+def test_pytorch_with_unsupported_version(use_loss_module=False):
+    smd.del_hook()
+
+    sim_class = SagemakerSimulator
+    with sim_class() as sim:
+        trainloader, testloader = get_dataloaders()
```

Comment:
Can we reuse the existing util functions instead of duplicating the training code? https://github.com/awslabs/sagemaker-debugger/blob/master/tests/zero_code_change/pt_utils.py#L67

Reply:
Done.